packages/devtools_app/assets/scripts/inspector_polyfill_script.dart

```diff
@@ -232,13 +241,16 @@ void addServiceExtensions() {
       // It is not a fatal error if some of the extensions fail to register
       // as could be the case if some are already defined directly within
       // package:flutter for the version of package:flutter being used.
-      print('Warning: unable to register service extension \'$name\'');
+      failures[name] = e.toString();
     }
   }
 
   registerHelper('getLayoutExplorerNode', getLayoutExplorerNode);
   registerHelper('setFlexFit', setFlexFit);
   registerHelper('setFlexFactor', setFlexFactor);
   registerHelper('setFlexProperties', setFlexProperties);
+  return failures.isNotEmpty
```

Comment:
why is this returning at all when the method signature return is `void`

Reply:
it isn't void. it is `String addServiceExtensions() {`

Reply:
oh whoops, missed a bracket.